manifests/config.pp

```diff
@@ -36,6 +36,11 @@
         mode    => '0664',
       }
 
+      file { "${::artifactory::artifactory_home}/etc/storage.properties":
```

Comment:
This symlink should mean the change to using `db.properties` doesn't break the module for artifactory 4 users.